doc/api-examples/rust/monitor-agent-connection.rs

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: MIT-0
+
+use dbus::{
+    blocking::{stdintf::org_freedesktop_dbus::PropertiesPropertiesChanged, Connection},
+    Message,
+};
+use std::time::Duration;
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let conn = Connection::new_system()?;
+
+    let bluechi_proxy = conn.with_proxy(
+        "org.eclipse.bluechi.Agent",
+        "/org/eclipse/bluechi",
+        Duration::from_millis(5000),
+    );
+    let _id = bluechi_proxy.match_signal(
+        |signal: PropertiesPropertiesChanged, _: &Connection, _: &Message| {
+            let changed_value = signal.changed_properties.get_key_value("Status").unwrap();
```

Comment:
Its just a demo, so I'm not sure how important this is, but it is generally frowned upon to just unwrap() things like this. It will panic the whole process if the property doesn't exist, and given that this is a network received thing we have no real guarantee that it does exist.